app/scripts/migrations/081.js

```diff
@@ -0,0 +1,68 @@
+import { cloneDeep } from 'lodash';
```

Comment:
Could you convert this to TypeScript? We have one TypeScript migration already, and the migration template is now in TypeScript as well.

Reply:
Yep! On it

Reply:
Done [here](https://github.com/MetaMask/metamask-extension/pull/17421/commits/0387d2ffdb9aafc899a8c8db4b23442f2e646a76)